Public/Src/Cache/DistributedCache.Host/Configuration/DistributedContentSettings.cs

```diff
@@ -215,6 +215,12 @@ private DistributedContentSettings(IDictionary<string, string> connectionSecretN
         [DataMember]
         public int EvictionPoolSize { get; set; } = 5000;
 
+        /// <summary>
+        /// A candidate must have an age older than this amount, or else it won't be evicted.
+        /// </summary>
+        [DataMember]
+        public TimeSpan EvictionMinAge { get; set; } = TimeSpan.FromMinutes(30);

```

Comment:
The feature should be disabled once it checked in.

It means that by default the time should be 0. Right?

Reply:
changed default to TimeSpan.Zero, configure value when we need to test for it.